src/pages/experiments/[experimentId]/data-processing/data-integration/components/DataIntegration.jsx

```diff
@@ -30,36 +48,153 @@ const defaultStylingConfig = {
-          </Tooltip>
+
+        <Col span={3}>
+          <Space direction='vertical'>
+            <Tooltip title='The number of dimensions used to configure the embedding is set here. This dictates the number of clusters in the Uniform Manifold Approximation and Projection (UMAP) which is taken forward to the ‘data exploration’ page.'>
+              <Button icon={<InfoCircleOutlined />} />
+            </Tooltip>
+
+            {Object.entries(plots).map(([key, plot]) => (
+              <button
+                type='button'
+                key={key}
+                onClick={() => setSelectedPlot(key)}
+                style={{
+                  padding: 0, margin: 0, border: 0, backgroundColor: 'transparent',
+                }}
+              >
+                <img
+                  alt={plot.title}
+                  src={plot.imgSrc}
+                  style={{
+                    height: '100px',
+                    width: '100px',
+                    align: 'center',
+                    padding: '8px',
+                    border: '1px solid #000',
+                  }}
+                />
+              </button>
+
+            ))}
+          </Space>
         </Col>
+
         <Col span={7}>
-          <CalculationConfig />
-          <PlotStyling
-            config={plotConfig}
-            onUpdate={updatePlotWithChanges}
-            singlePlot
-          />
+          <Collapse defaultActiveKey={['data-integration']}>
+            <Panel header='Data Integration' key='data-integration'>
+              <CalculationConfig experimentId={experimentId} />
+            </Panel>
+            <Panel header='Plot Styling' key='styling'>
+              <Collapse accordion>
+                <Panel header='Main Schema' key='main-schema'>
+                  <DimensionsRangeEditor
+                    config={config}
+                    onUpdate={updatePlotWithChanges}
+                  />
+                  <Collapse accordion>
+                    <Panel header='Define and Edit Title' key='title'>
+                      <TitleDesign
+                        config={config}
+                        onUpdate={updatePlotWithChanges}
+                      />
+                    </Panel>
+                    <Panel header='Font' key='font'>
+                      <FontDesign
+                        config={config}
+                        onUpdate={updatePlotWithChanges}
+                      />
+                    </Panel>
+                  </Collapse>
+                </Panel>
+                <Panel header='Axes and Margins' key='axes'>
+                  <AxesDesign config={config} onUpdate={updatePlotWithChanges} />
+                </Panel>
+                <Panel header='Colours' key='colors'>
+                  <ColourInversion
+                    config={config}
+                    onUpdate={updatePlotWithChanges}
+                  />
+                  {(selectedPlot === 'sample' || selectedPlot === 'frequency') ? (
```

Comment:
on line 171 you have the same conditions, maybe merge those ternary operators into a single one?

Reply:
The conditions are located under different `<Panel />` scope. If I were to group them, there would be some chunks of duplicated code :

```
{(selectedPlot === 'sample' || selectedPlot === 'frequency') ? (
  <Panel header='Colours' key='colors'>
   <ColourInversion>
      <Alert ... />
   </ColourInversion>
  </Panel>
<Panel header='Legend' key='legend'>
                    <LegendEditor ... />
                  </Panel>
) : (
<Panel header='Colours' key='colors'>
   <ColourInversion />
  </Panel>
)
```

By writing this way, I try to minimize code duplication, but as you can see, it comes with more if statements. Some of this is a matter of personal preference, but I think each has its own merits. Maybe @ivababukova and @marcellp can also weigh in on this?

Reply:
Since there are lots of if statements here, the way I would do it is to first define the three different views, one for sample, one for elbow and one for frequency. These can be put in the `plots` object, so something like this:

```
const plots = {
    sample: {
      title: 'Samples',
      imgSrc: cellBySamplePic,
      stylingSpecifics: (config) => (<Panel header='Colours' key='colors'>
   <ColourInversion>
      <Alert ... />
   </ColourInversion>
  </Panel>
<Panel header='Legend' key='legend'>
                    <LegendEditor ... />
                  </Panel>)
    },
```
and then in here, I would just plug in the styling specifics for the relevant plot

Reply:
Factored out into plot-specific option under `plots`